sidekick/sidekick-broker/sidekick-broker-app/src/main/java/com/runsidekick/broker/SidekickBrokerApplication.java

```diff
@@ -15,7 +19,13 @@
 )
 @RestController
 @EnableScheduling
-public class SidekickBrokerApplication {
+public class SidekickBrokerApplication implements CommandLineRunner {
+
+    @Value("${phoneHome.enabled:true}")
```

Comment:
`phonehome` (no camel case) might better while converting env vars to config names